src/adapter/z-stack/znp/buffaloZnp.ts

```diff
@@ -146,20 +154,163 @@ class BuffaloZnp extends Buffalo {
         return value;
     }
 
-    public read(type: string, options: BuffaloZnpOptions): TsType.Value {
-        if (type === 'LIST_ROUTING_TABLE') {
-            return this.readListRoutingTable(options);
-        } else if (type === 'LIST_BIND_TABLE') {
-            return this.readListBindTable(options);
-        }  else if (type === 'LIST_NEIGHBOR_LQI') {
-            return this.readListNeighborLqi(options);
-        } else if (type === 'LIST_NETWORK') {
-            return this.readListNetwork(options);
-        } else if (type === 'LIST_ASSOC_DEV') {
+    // eslint-disable-next-line @typescript-eslint/no-explicit-any
+    public write(type: ParameterType, value: any, options: BuffaloZnpOptions): void {
+        switch (type) {
```

Comment:
This looks like a lot of code duplication here, why not use `return super.read(type, options);`?

Reply:
I removed the `read`/`write` from the base class since it was forcing to use strings (not the actual enums), aliases, etc to "allow" `super.xyz` in a generic fashion (and was forcing comparisons with types irrelevant to stacks in many cases).
Basically, it's written in code directly now, instead of parsing alias "matcher" objects and various tricks at runtime that made it work before.
It also ensures that each type actually required by a stack, based on associated enum, is covered (TS narrowing/Jest).